Automattic/wp-calypso
# Theme Showcase: Refactor translatable title and description with hooks

Related to 478-gh-Automattic/i18n-issues#issuecomment-1678568668

## Proposed Changes

* Refactor theme showcase title and description logged-in and logged-out content to use custom hooks so that the component is properly subscribed to i18n changes and updates when the i18n data changes.

## Testing Instructions



* Code review.
* Use calypso.live build.
* Go to `/[locale]/themes` logged-out, where `[locale]` is a locale slug for any Mag-16 language.
* Confirm title and meta tags are rendered translated into the specified language.
* Go to `/themes` logged-in.
* Confirm title and description are translated in your account language.
* Unit tests for logged-in title and description pass - `yarn test-client client/my-sites/themes/test/use-theme-showcase-*`.

## Pre-merge Checklist
